src/components/RadioButton/RadioButton.st.css

```diff
@@ -0,0 +1,182 @@
+
+:import {
+    -st-from: "wix-ui-core/index.st.css";
+    -st-named: RadioButton;
+}
+
+:import {
+    -st-from: "../../common/palette.st.css";
+    -st-named: errorColor;
+}
+
+/*Defaults*/
+:vars {
+    DefaultTextColor: color-5;
+    DefaultBorderColor: color-5;
+    DefaultIconColor: color-8;
+    DisabledColor: gray;
+}
+
+/*Overrides*/
+:vars {
+    TextColor: --overridable;
+    BorderColor: --overridable;
+    IconColor: color-8;
+}
+
+:vars {
+    CurrentBorderColor: color(fallback(value(BorderColor), value(DefaultBorderColor)));
+    CurrentIconColor: color(fallback(value(IconColor), value(DefaultIconColor)));
+    CurrentTextColor: color(fallback(value(TextColor), value(DefaultTextColor)));
+    LabelTextFont: "{theme: 'Body-M', size: '16px', lineHeight: '1.5em'}";
+}
+
+.labelWrapper {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+}
+
+.root {
+    -st-extends: RadioButton;
+    -st-states: checked, disabled, box, error;
+    display: flex;
+    line-height: 20px;
+}
+
+.root:error .checkmark {
+    border-color: color(value(errorColor));
+}
+
+.root:box:disabled {
+    cursor: default;
+}
+
+.root:box:hover, .root:box:focus-within {
+    border-color: applyOpacity(value(CurrentBorderColor), 1);
+}
+
+.root:box.disabled:hover:not(:checked), .root:box:disabled:focus-within:not(:checked) {
+    border-color: applyOpacity(value(CurrentBorderColor), 0.6);
+}
+
+.root:box:checked {
+    background-color: applyOpacity(value(CurrentIconColor), 0.1);
+    border-color: value(CurrentIconColor);
+}
+
+.root:box:checked:not(:disabled):hover, .root:box:checked:not(:disabled):focus-within {
+    border-color: applyOpacity(value(CurrentIconColor), 0.6);
+}
+
+.root:box:not(:disabled):hover .icon {
+    border-color: applyOpacity(value(CurrentBorderColor), 1);
+}
+
+.root::label {
+    width: 100%;
+    font: font(value(LabelTextFont));
+    color: applyOpacity(value(CurrentTextColor),1);
+}
+
+.root::icon {
+    display: flex;
+    position: relative;
+    margin-bottom: 1px;
+    top:3px;
+    cursor: pointer;
+    font-size: 22px;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+}
+
+.root::icon:before {
+    content: '';
+    width: 35px;
+}
+
+.checkmark {
+    position: absolute;
+    top: 0;
+    background-color: transparent;
+    left: 0;
+    right: 0;
+    height: 20px;
+    border-radius: 50%;
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 20px;
+    border: 1px solid applyOpacity(value(CurrentBorderColor), 0.6);
+}
+
+.innerCheck {
+    margin: 2px;
+    width: 10px;
+    height: 10px;
+    border-radius: 50%;
+    transform: scale(0.5);
+    opacity: 0;
+    transition: transform .2s ease-in-out, opacity .2s ease-in-out;
+}
+
+
+.root:box {
+    display: flex;
+    padding: 16px;
+    margin-bottom: 23 [x];
+    background-color: color(color-1);
```

Comment:
The background color should be taken from a variable, so that it can be overridden.
See the specs in zeroheight